helm/dagster/charts/dagster-user-deployments/templates/helpers/_helpers.tpl

```diff
@@ -148,5 +148,9 @@ DAGSTER_K8S_PIPELINE_RUN_ENV_CONFIGMAP: "{{ template "dagster.fullname" . }}-pip
       pod_template_spec_metadata:
         annotations: {{- toYaml .annotations | nindent 10 }}
       {{- end }}
+      {{- if .tolerations }}
+      pod_template_spec_metadata:
+        tolerations: {{- toYaml .tolerations | nindent 10 }}
+      {{- end }}
```

Comment:
this is setting `pod_template_spec_metadata` twice if both annotations and tolerations are set, which isn't valid YAML